src/validation.cpp

```diff
@@ -4204,7 +4204,7 @@ VerifyDBResult CVerifyDB::VerifyDB(
     bool skipped_l3_checks{false};
     LogPrintf("Verification progress: 0%%\n");
 
-    const bool is_snapshot_cs{!chainstate.m_from_snapshot_blockhash};
+    const bool is_snapshot_cs{chainstate.m_from_snapshot_blockhash};
```

Comment:
In commit "bugfix: correct is_snapshot_cs in VerifyDB" (b85530c18bfe24ff434f32f959d8dd843afca54d)

Would it make sense to put this fix or other fixes into a backport PR? If this PR were merged and released with Bitcoin Core 26, would it be possible for someone to load an assumeutxo snapshot with Bitcoin Core 26, then downgrade to Bitcoin Core 25 or 24 and run into bugs processing the snapshot chainstate?

Reply:
My inclination is to say that if the user begins snapshot use with a client that has this changeset, they won't be able to downgrade without issue until the snapshot is validated (based on e.g. the net_processing changes here). I'm not sure where the best place to convey that would be... release notes?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/27596#discussion_r1312047335

> My inclination is to say that if the user begins snapshot use with a client that has this changeset, they won't be able to downgrade without issue until the snapshot is validated (based on e.g. the net_processing changes here). I'm not sure where the best place to convey that would be... release notes?

I think the most helpful place would be the `loadtxoutset` RPC documentation, saying that using the  `loadtxoutset`  RPC will make it temporarily not possible to downgrade the node software until the snapshot is validated, and then after that it will be safe to downgrade again.

I wonder if we can do something simple to avoid this problem, though, like tweaking the snapshot chainstate directory name before releasing, so old software ignores it, or backporting a patch to the stable branches that  makes them ignore the directory.